src/main/pages/main/CompatibleDevices.js

```diff
@@ -0,0 +1,195 @@
+import React, {useEffect, useState} from "react";
+import {observer} from "mobx-react";
+import Papa from "papaparse";
+import {mainStore} from "../../stores/Main";
+import DeviceCSV from "../../static/documents/devices.csv";
+import {SearchIcon} from "../../static/icons/Icons";
+import ImageIcon from "../../components/ImageIcon";
+import DevicesGraphic from "../../static/images/main/media_wallet/devices-graphic.png";
+import {PageContainer} from "../../MainApp";
+
+const AutocompleteResults = ({
+  searchTerm,
+  results,
+  setSearchTerm,
+  setResults,
+  hasMatch=false,
+  selectedItemIndex
+}) => {
+  if(!searchTerm) { return null; }
+
+  const ResultsHeader = () => {
+    if(!results.length) {
+      return "Sorry, this platform isn't supported yet.";
+    } else if(hasMatch) {
+      return `${results[0]} is supported.`;
+    }
+  };
+
+  return (
+    <div className="compatible-devices__autocomplete-results">
+      {
+        (!results.length || hasMatch) ?
+          <div className="compatible-devices__autocomplete-results">
+            <h3 className="compatible-devices__autocomplete-results-header">
+              {ResultsHeader()}
```

Comment:
why define this as a separate function?

Reply:
ResultsHeader() was bigger before.. I'll remove the separation